pypeit/core/flexure.py

```diff
@@ -25,6 +28,10 @@
 from pypeit.core import arc
 from pypeit.core import qa
 from pypeit.core import fitting
+from pypeit.datamodel import DataContainer
+from pypeit import specobjs
+
+from pypeit.spectrographs import keck_deimos
```

Comment:
Please remove this line. Specific spectrographs should never be imported in this way.

Reply:
agreed.  wasn't used either